data/json/items/archery.json

```diff
@@ -2,258 +2,247 @@
   {
     "type": "AMMO",
     "id": "arrow_fire_hardened_fletched",
-    "price": 900,
-    "name": "fire-hardened wooden arrow",
+    "price": 100,
+    "name": "crude wooden arrow",
     "symbol": "=",
     "color": "green",
-    "description": "This simple arrow has a fire-hardened point and fletching.  Stands a below average chance of remaining intact once fired.",
+    "description": "A crude pointed wooden shaft with a notch at the back.  Stands a very low chance of remaining intact once fired.",
     "material": "wood",
     "volume": 1,
-    "weight": 43,
+    "weight": 30,
     "bashing": 3,
     "ammo_type": "arrow",
-    "damage": 4,
-    "range": 6,
-    "dispersion": 190,
+    "prop_damage": 0.7,
+    "range": -4,
+    "dispersion": 250,
     "loudness": 0,
     "count": 10,
-    "effects": [ "RECOVER_5" ]
+    "effects": [ "RECOVER_2" ]
   },
   {
     "type": "AMMO",
     "id": "arrow_field_point_fletched",
     "price": 1000,
-    "name": "field point wooden arrow",
+    "name": "wooden bodkin arrow",
     "symbol": "=",
     "color": "green",
-    "description": "A simple arrow shaft with a crude arrowhead and fletching.  Useful for hunting small woodland creatures, or as a last ditch defense against zombies.  Stands a below average chance of remaining intact once fired.",
-    "material": "wood",
+    "description": "A fletched wooden arrow shaft with a pointed tip.  Useful for piercing armor.  Stands a decent chance of remaining intact once fired.",
+    "material": [ "wood", "steel" ],
     "volume": 1,
-    "weight": 46,
+    "weight": 55,
     "bashing": 3,
     "ammo_type": "arrow",
-    "damage": 7,
-    "pierce": 2,
-    "range": 6,
-    "dispersion": 175,
+    "prop_damage": 1.12,
+    "pierce": 3,
+    "dispersion": 100,
     "loudness": 0,
     "count": 10,
-    "effects": [ "RECOVER_5" ]
+    "effects": [ "RECOVER_30" ]
   },
   {
     "type": "AMMO",
     "id": "arrow_small_game_fletched",
-    "price": 600,
-    "name": "small game arrow",
+    "price": 300,
+    "name": "blunt simple wooden arrow",
     "symbol": "=",
     "color": "green",
-    "description": "A simple arrow shaft that has a wide, blunt head.  Useful for hunting small woodland creatures without splattering them all over the ground, but stands a below average chance of remaining intact once fired.",
+    "description": "A simple fletched wooden arrow shaft with a blunted tip.  Useful for hunting small woodland creatures without splattering them all over the ground.  Stands a low chance of remaining intact once fired.",
     "material": "wood",
     "volume": 1,
-    "weight": 44,
+    "weight": 35,
     "bashing": 4,
     "ammo_type": "arrow",
-    "damage": 3,
-    "range": 6,
+    "prop_damage": 0.5,
+    "range": -6,
     "dispersion": 210,
     "loudness": 0,
     "count": 10,
-    "effects": [ "RECOVER_5", "NOGIB" ]
+    "effects": [ "RECOVER_10", "NOGIB" ]
```

Comment:
Now that this is not a game arrow, does this no longer need NOGIB?

Reply:
While it isn't explicitly a small game arrow, the blunt tip serves the same purpose. The description reflects this, so hopefully players won't get confused.

That said, a constant naming convention would be a good idea.